opentok/opentok.py

```diff
@@ -535,6 +536,8 @@ def start_archive(
         StreamModes.manual to explicitly select streams to include in the the archive, using the
         OpenTok.add_archive_stream() and OpenTok.remove_archive_stream() methods.
 
+        :param String multi_archive_tag (Optional): If specified, triggers a new simultaneous archive on the session. 
```

Comment:
Have changed the docs. The `multi_archive_tag` class attribute would not be present in the `Archive` class because it's not returned by OpenTok or used in this way.